src/org/ods/component/Pipeline.groovy

```diff
@@ -355,26 +356,34 @@ class Pipeline implements Serializable {
 
             logger.info 'Environment does not exist yet. Creating now ...'
             script.withCredentials(
-                [script.usernameColonPassword(credentialsId: context.credentialsId, variable: 'USERPASS')]
+                [script.usernamePassword(
+                    credentialsId: context.credentialsId,
+                    usernameVariable: 'USERNAME',
+                    passwordVariable: 'PASSWORD')
+                ]
             ) {
-                def userPass = script.USERPASS.replace('$', '\'$\'')
                 def branchName = "${script.env.JOB_NAME}-${script.env.BUILD_NUMBER}-${context.cloneSourceEnv}"
                 logger.info "Calculated branch name: ${branchName}"
                 def scriptToUrls = context.getCloneProjectScriptUrls()
                 // NOTE: a for loop did not work here due to https://issues.jenkins-ci.org/browse/JENKINS-49732
                 scriptToUrls.each { scriptName, url ->
-                    script.sh(script: "curl --fail -s --user ${userPass} -G '${url}' -d raw -o '${scriptName}'")
+                    script.echo "curl --fail -s -G '${url}' -d raw -o '${scriptName}'"
+                    script.sh(script: """set +x
+                        curl --fail -s \\
+                        --header \"${AuthUtil.header(AuthUtil.Scheme_Basic, script.USERNAME, script.PASSWORD)}\" \\
```

Comment:
Can you make this `script.env.USERNAME` etc.? I think reading from `env` is the "right way" - and it allows to use an interface for `script` (`IPipelineSteps`).